relayer/cmd/generate_beacon_data.go

```diff
@@ -266,32 +237,7 @@ func generateBeaconData(cmd *cobra.Command, _ []string) error {
 		writeJSONToFile(headerUpdate, fmt.Sprintf("execution-header-update.%s.json", activeSpec.ToString()))
 		log.Info("created execution update file")
 
-		if activeSpec.IsMinimal() {
-			// generate FinalizedUpdate for next period
-			log.Info("waiting until next sync period,several minutes required...")
-			time.Sleep(time.Duration(specSettings.SlotsInEpoch*(specSettings.EpochsPerSyncCommitteePeriod-elapseEpochs)) * SlotTimeDuration)
-			nextFinalizedUpdateScale, err := s.GetFinalizedUpdate()
-			if err != nil {
-				return fmt.Errorf("get next finalized header update: %w", err)
-			}
-			nextFinalizedUpdate := nextFinalizedUpdateScale.Payload.ToJSON()
-			nextFinalizedUpdatePeriod := s.ComputeSyncPeriodAtSlot(nextFinalizedUpdate.FinalizedHeader.Slot)
-			if initialSyncPeriod+1 != nextFinalizedUpdatePeriod {
-				return fmt.Errorf("nextFinalizedUpdatePeriod should be 1 period ahead of initialSyncPeriod")
-			}
-			writeJSONToFile(nextFinalizedUpdate, fmt.Sprintf("next-finalized-header-update.%s.json", activeSpec.ToString()))
-			log.Info("created next finalized header update file")
-
-			// generate nextSyncCommitteeUpdate
-			nextSyncCommitteeUpdateScale, err := s.GetSyncCommitteePeriodUpdate(initialSyncPeriod + 1)
-			if err != nil {
-				return fmt.Errorf("get sync committee update: %w", err)
-			}
-			nextSyncCommitteeUpdate := nextSyncCommitteeUpdateScale.Payload.ToJSON()
-			writeJSONToFile(nextSyncCommitteeUpdate, fmt.Sprintf("next-sync-committee-update.%s.json", activeSpec.ToString()))
-			log.Info("created next sync committee update file")
-		}
-
+		// Generate benchmark fixture for mainnet spec only
```

Comment:
How would we regenerate the minimal spec test fixture data? 🤔

Reply:
Yes, I think it's fine to also generate benchmark fixtures for minimal spec, then we can avoid the conditional add_benchmarks hack in the Rococo runtime.

https://github.com/Snowfork/snowbridge/pull/1097/commits/63a1bcdb9a432abaa0d25d564375694242f1d26b